doc-sphinx/source/user_guide/launching.rst

```diff
@@ -1,45 +1,39 @@
-=====================================
-Launching the OLCF Test Harness (OTH)
-=====================================
+.. _section_launching_oth:
 
-To launch the OLCF Test Harness (OTH) you must first access the harness code.
-This can be done in two ways, one by obtaining your own copy of the code or
-using the centralized harness code that will be available on the system under
-test.
-
-OTH Setup
----------
+=================
+Quick-Start Guide
+=================
 
-Option 1: Using the centralized (pre-built) OTH
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+This section serves as a quick-start guide for installing the OTH and launching an existing test.
+For creating a new test, see :ref:`section_new_test`.
+For adding support for a new machine, see :ref:`section_new_machine`.
 
-On Lyra
-"""""""
+.. _oth_setup:
 
-The code is already installed in: */sw/lyra/acceptance/olcf-test-harness*
+Installation
+------------
 
-Setup the environment:
-
-.. code-block:: bash
+To launch the OLCF Test Harness (OTH) you must first access the harness code.
+This can be done in two ways: by obtaining your own copy of the code or using the centralized harness code that will be available on each system under test.
```

Comment:
I think the a path is missing at the end of the sentence?

Reply:
Wait, no, I understand now, I think. Brain was not firing on all cylinders. 

It might be clearer to say something like "centralized harness code that is available on most OLCF systems."?